configure.bat

```diff
@@ -22,6 +22,13 @@ IF %TARGET_ARCH%==x86_64 (
   SET MSVC_ARCH=%TARGET_ARCH%
 )
 
+REM The LiveCode name for x86_64 is x86_64
+IF %MSVC_ARCH%==x64 (
+    SET LC_ARCH=x86_64
+) ELSE (
+    SET LC_ARCH=x86
+)
+
```

Comment:
There are still things which rely on TARGET_ARCH being the (native) arch name. This needs a bit of a moomin - the current configure.bat ensures that MSVC_ARCH and TARGET_ARCH are both set to X64 for 64-bit, where as LC_ARCH uses X86_64.